### PR TITLE
support aarch64

### PR DESCRIPTION
**Pull request by jefby**
support aarch64 platform 

Signed-off-by: jefby <jef199006@gmail.com>

### CODE REVIEW

**Comment by vpodzime** [comment]
Is this really enough to make things actually work?

**Comment by jefby** [comment]
@vpodzime  Yes,i already use it to create a centos7-arm64 docker image

**Comment by vpodzime** [comment]
Great! This looks good to me then.

File: imgcreate/live.py [hunk 1]
```diff
@@ -1035,6 +1035,8 @@ def _get_excluded_packages(self):
     LiveImageCreator = ppc64LiveImageCreator
 elif arch.startswith('arm'):
```
Comment (FGrose) [inline imgcreate/live.py]: Why not
`elif arch.startswith(('arm', 'aarch64')):`
?

Comment (jefby) [inline imgcreate/live.py]: @FGrose  Yeah, i think this is more perfect.

**Comment by bcl** [comment]
fixed and pushed, thanks!

**Pull request closed without merging**